src/procedures/__tests__/SetDividendsWallet.ts

```diff
@@ -177,5 +227,51 @@ describe('SetDividendsWallet', () => {
       ).toEqual(true);
       expect(addTransactionSpy.callCount).toEqual(1);
     });
+
+    test('should successfully refresh ERC20 dividends factory', async () => {
+      const refreshStub = erc20FactoryMock.mock(
+        'refresh',
+        Promise.resolve(undefined)
+      );
+      const resolverValue = setDividendsWalletModule.createSetDividendWalletResolver(
```

Comment:
This should be awaited here to make sure there is no chance for the stubs to not have been called by the time the verifications are made